Sources/AppStoreConnectCLI/Commands/Certificates/CreateCertificateCommand.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Combine
+import Foundation
+
+struct CreateCertificateCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "create",
+        abstract: "Create a new certificate using a certificate signing request.")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(help: "The type of certificate to create \(CertificateType.allCases).")
+    var certificateType: CertificateType
```

Comment:
Here I'm using `CertifcateType` directly because I made it `ExpressibleByArgument`.  We use this technique elsewhere in the code.